packages/network/test/unit/cors_spec.ts

```diff
@@ -3,46 +3,44 @@ import { expect } from 'chai'
 
 describe('lib/cors', () => {
   context('.parseUrlIntoDomainTldPort', () => {
-    beforeEach(function () {
-      this.isEq = (url, obj) => {
-        expect(cors.parseUrlIntoDomainTldPort(url)).to.deep.eq(obj)
-      }
-    })
+    const assertParsesUrlCorrectly = (url, obj) => {
```

Comment:
nit: named assert but uses expect

Reply:
updated to `expectUrlToBeParsedCorrectly` in a2637b9a76